tests/test_dask_image/test_ndinterp/test_affine_transformation.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+
+import dask_image.ndinterp as da_ndinterp
+
+import numpy as np
+import dask.array as da
+from scipy import ndimage
+
+
+class Helpers:
+    @staticmethod
+    def test_affine_transform(n=2,
```

Comment:
A class with a single static method probably would be simpler if it was a single function. 

If you call that function something like `validate_affine_transform` (something without the `test_` prefix so pytest doesn't pick it up) then you can get rid of both the `Helper` class and the pytest fixture in one go.

Reply:
Oh I wasn't aware that changing the prefix was enough 😅. Thanks!